File: ledblink.py
import RPi.GPIO as GPIO
from time import sleep
GPIO.setmode(GPIO.BCM)
GPIO.setup(17, GPIO.OUT)
print "All Set in Python! Let's Blink"
for i in range(1, 1000):
	GPIO.output(17, GPIO.HIGH)
	sleep(0.1)
	GPIO.output(17, GPIO.LOW)
	sleep(0.1)
GPIO.cleanup()

File: ledblink.c
#include<stdio.h>
#include<wiringPi.h>
#include<time.h>

void main() {
	wiringPiSetup();
	pinMode(17, OUTPUT);
	printf("\n Hi, I am using wiringPi, Blink Blink!");
	printf("\n Press ctrl C to exit\n");
	for(;;) {
		digitalWrite(17, HIGH);
		delay(1000);
		digitalWrite(17, LOW);
		delay(500);
	}
}
